### PR TITLE
Update test

### PR DESCRIPTION
**Pull request by ricardogobbosouza**
- Remove `get-port` of `devDependencies`
- Use `setupNuxt`

### CODE REVIEW

File: test/module.test.js [hunk 1]
```diff
@@ -5,7 +5,6 @@ const request = require('request-promise-native')
 const consola = require('consola')
 
 const config = require('./fixture/nuxt.config')
-config.dev = false
```
Comment (pi0) [inline test/module.test.js]: Why this line removed?

Comment (ricardogobbosouza) [reply test/module.test.js]: It's indifferent to this module

Comment (pi0) [reply test/module.test.js]: Default value of dev is from `std-env.dev`:

https://github.com/nuxt/nuxt.js/blob/dev/packages/config/src/config/_common.js#L6

Which is based on `NODE_ENV` environment variable:

https://github.com/jsless/std-env/blob/master/index.js#L50

I know not different but as a best practice would you please revert it back? Be ensure is always better :)

Comment (ricardogobbosouza) [reply test/module.test.js]: Ok :)

**Pull request merged**